GCTC-NTGC/gc-digital-talent
# Connect edit profile pages to application

Resolves #3327.

Changes links and breadcrumbs for profile edit pages if application query parameter is provided.

To test:
- Visit each of the profile edit pages without an 'application' query parameter and that they appear the same as before.
- Visit them each again with a valid 'application' query parameter and ensure their breadcrumbs are different and links point to the application page.
